test/unittest/tests/wrappers/connection_wrapper.cpp

```diff
@@ -16,7 +16,20 @@
 ConnectionWrapper::ConnectionWrapper(const std::string &db_home)
     : _conn_impl(nullptr), _conn(nullptr), _db_home(db_home)
 {
-    utils::throwIfNonZero(mkdir(_db_home.c_str(), 0700));
+    struct stat sb;
```

Comment:
What problem is being solved by checking whether the directory exists? It should (in theory) be cleaned up by the destructor.